xapi-profiles-structure.md

```diff
@@ -452,12 +463,17 @@ Property | Type | Description | Required
 
 A Statement Template MUST NOT have both `objectStatementRefTemplate` and `objectActivityType`.
 
-The verb, object activity type, attachment usage types, and context activity types listed are called Determining Properties.
+The verb, object activity type, homepage, extensions, category context activity, attachment usage types, and context activity types listed are called Determining Properties.
 
-A Profile Author MUST change a Statement Template's `id` between versions if any of the Determining Properties, StatementRef properties, or rules change. Changes of `scopeNote` are not considered changes in rules.
+A Profile Author MUST change a Statement Template's `id` between versions if any of the Determining Properties, StatementRef properties, or rules change. 
+Changes of `scopeNote` are not considered changes in rules.
+A Profile Author SHOULD create a Statement Template with a single Determining Property of the Category Context Activity
+Other Statement Templates MAY use the Category Context Activity of the intended xAPI Profile in combination with another Determining Property. 
+In previous versions of this document, and if a Statement Template with a single Determining Property of the Category Context Activity is not included, All Statement 
+Templates associated with this profile MUST consider that Category Context Activity as a Determining Property as it was an implicit requirement in previous versions.  
```

Comment:
I think it may be easier to make the profile version reference field required. Each Statement Template must include an array that includes at least the current profile version iri. Other Profile Version iris can be included but are not necessary

Reply:
6/7/2022 - Getting at the notion that we are making it an array as opposed to just the singular value.